client-participation/package.json

```diff
@@ -12,21 +12,32 @@
   },
   "dependencies": {
     "autosize": "^3.0.3",
+    "backbone": "github:pol-is/backbone#polis",
     "boolean": "^0.1.3",
+    "bootstrap": "^3.1.1",
+    "bootstrap-sass": "^3.2.0",
     "brain": "~0.6.3",
     "combine-css": "0.0.2",
+    "d3": "^3.5.17",
     "d3-force": "^1.2.1",
+    "d3-tip": "^0.6.7",
+    "d3-v4": "npm:d3@^4.8.0",
```

Comment:
Noting that we have two versions of d3 right now. This is probably an intentional transition step.

Reply:
heh as i understand, it was when bringing in the new "vis2" (built on react) without breaking the old backbone vis (and maybe other stuff) that used old d3. My sense is that we could prob tackle removing v3 soon :)